src/amo/components/SearchFilters/index.js

```diff
@@ -196,6 +214,90 @@ export class SearchFiltersBase extends React.Component {
             </div>
           )}
 
+          {filters.addonType == getAddonTypeFilter(ADDON_TYPE_THEME) && (
+            <div>
+              <label
+                className="SearchFilters-label SearchFilters-Color-label"
+              >
+                {i18n.gettext('Theme Color')}
+              </label>
+              <ul className="SearchFilters-ThemeColors">
+                <li
```

Comment:
I suggest you create a config array and loop over it:

```js
const themeColors = [
  {
    id: 'black',
    color: '000000',
  },
  // ...
];
```

```js
{themeColors.map((themeColor) => (
  <li
    key={themeColor.id}
    className={makeClassName(
      'SearchFilter-ThemeColors-ColorItem',
      `SearchFilter-ThemeColors-ColorItem--${themeColor.id}`,
      {
        'SearchFilter-ThemeColors-ColorItem--active': filters.color == themeColor.color,
      },
    )}
    onClick={this.onColorClick}
  />
))}
```

Reply:
Definitely yeah. Do you have a suggestion regarding how to refactor the css as well ? Not too familiar with sass, I could look it up but maybe you already know :)

Reply:
Switching to css classes would be great, but then I don't think you can DRY more. There is the concept of [sass maps](https://www.sitepoint.com/using-sass-maps/) but I don't think it will make you write more readable code.